modules/ocf_irc/manifests/init.pp

```diff
@@ -1,4 +1,11 @@
 class ocf_irc {
   include ocf_ssl
+  include ocf_irc::ircd
+  include ocf_irc::services
   include ocf_irc::slack
+
+  # Make the irc user able to read the certs for running the IRCd with SSL
+  user { 'irc':
+    groups => 'ssl-cert',
+  }
```

Comment:
You might wish to `require => Package['whatever-package-adds-this-user']` here so that the groups are added after the user is created, rather than creating a user here.

Otherwise, on the initial Puppet run, this user might get created with some default home directory, groups, shell, etc., and the package install won't re-create the user or set the settings properly.